www/templates/html/PersonInfo/SignatureGenerator.tpl.php

```diff
@@ -0,0 +1,890 @@
+                        c0.4,0,0.8-0.1,1.1-0.4l9.9-9.9l9.9,9.9c0.3,0.3,0.7,0.4,1.1,0.4c0,0,0,0,0,0c0.4,0,0.8-0.2,1.1-0.4c0.3-0.3,0.4-0.7,0.4-1.1
+                        C25,23.1,24.8,22.7,24.6,22.4z"/>
+                </svg>
+            </button>
+            <div class="dcf-d-flex dcf-flex-col dcf-row-gap-3">
+                <button class="up-field-btn dcf-btn dcf-btn-secondary dcf-p-2" aria-label="Move Field Up" type="button">
+                    <svg xmlns="http://www.w3.org/2000/svg" class="dcf-fill-current dcf-w-4 dcf-h-4 dcf-d-block" viewBox="0 0 25 26">
+                        <path d="M24.6,13L13.7,0.6l-0.1-0.2l-0.1,0c-0.5-0.4-1.4-0.4-1.9,0
+                            L0.4,13C0.1,13.3,0,13.7,0,14.1c0,0.4,0.2,0.8,0.5,1
+                            c0.6,0.5,1.6,0.5,2.1-0.1L11,5.5v19c0,0.8,0.7,1.5,1.5,1.5
+                            s1.5-0.7,1.5-1.5v-19l8.4,9.5c0.5,0.6,1.5,0.7,2.1,0.1
+                            c0.3-0.3,0.5-0.6,0.5-1C25,13.7,24.9,13.3,24.6,13z"/>
+                    </svg>
+                </button>
+                <button class="down-field-btn dcf-btn dcf-btn-secondary dcf-p-2" aria-label="Move Field Down" type="button">
+                    <svg xmlns="http://www.w3.org/2000/svg" class="dcf-fill-current dcf-w-4 dcf-h-4 dcf-d-block" viewBox="0 0 24 24">
+                        <svg xmlns="http://www.w3.org/2000/svg" class="dcf-fill-current dcf-w-4 dcf-h-4 dcf-d-block" viewBox="0 0 25 26">
+                            <path d="M24.5,10.9c-0.3-0.3-0.7-0.4-1.1-0.4c-0.4,0-0.8,0.2-1,0.5
+                                L14,20.5v-19C14,0.7,13.3,0,12.5,0S11,0.7,11,1.5v19L2.6,11
+                                c-0.3-0.3-0.6-0.5-1-0.5c-0.4,0-0.8,0.1-1.1,0.4c-0.3,0.3-0.5,0.6-0.5,1
+                                c0,0.4,0.1,0.8,0.4,1.1l11,12.4l0.1,0.2l0.1,0.1
+                                c0,0,0.1,0.1,0.2,0.1l0.2,0.1c0.2,0.1,0.4,0.1,0.6,0.1
+                                c0.2,0,0.4,0,0.6-0.1l0.2-0.1c0.1,0,0.1-0.1,0.2-0.1L24.6,13
+                                c0.3-0.3,0.4-0.7,0.4-1.1C25,11.5,24.8,11.1,24.5,10.9z"/>
+                        </svg>
+                </button>
+            </div>
+        </div>
+    </li>
+</template>
+
+<template id="signature-primary-field-template">
+    <p style="margin:0px 2px 2px 2px;font-family: Helvetica, Arial;color: #454545;font-size:16px;line-height: 18px;"></p>
+</template>
+
+<template id="signature-secondary-field-template">
+    <p style="font-size:12px;margin:2px;font-style:italic;font-family: Georgia;color: #454545;line-height: 14px;"></p>
+</template>
+
+<template id="signature-field-template">
+    <p style="font-size:10px;margin:2px;font-family: Helvetica, Arial;color: #454545;line-height: 14px;"></p>
+</template>
+
+<script>
+    const default_person_to_load = '<?php echo $context->getUser(); ?>';
+</script>
+
+<script>
+    const logo_url = '<?php echo UNl_Peoplefinder::$url . 'images/logos/'; ?>';
```

Comment:
Can you move the declaration of `logo_url` to be in the same script tag as `default_person_to_load`